privacyidea/lib/tokens/pushtoken.py

```diff
@@ -729,6 +729,32 @@ def create_challenge(self, transactionid=None, options=None):
                                                      challenge, fb_gateway,
                                                      pem_privkey, options)
             res = fb_gateway.submit_message(self.get_tokeninfo("firebase_token"), smartphone_data)
+
+            # Create the challenge in the challenge table regardless of a failure
```

Comment:
The if-statement actually now means s.th. like

Create the challenge in the challenge table if either the messages was successfully submitted
to the Firebase API or if polling is allowed in general or for this specific token.

Reply:
Done in 012b6af7